### ISSUE TITLE
Continue to Display Last Played Track's Lyrics

### ISSUE THREAD

**Issue by JayJonahJaymeson** [jacquesh/foo_openlyrics#234]
Is it possible to set OpenLyrics so that when a track stops it continues to display that track's lyrics until another is started?

**Comment by jacquesh** [comment]
No, if you pause a track then it will continue showing that track's lyrics, but if you stop playback entirely then it will always stop showing lyrics.

One reason for this is that it's not at all clear what timestamp should be used when showing lyrics if there's no track "playing" (paused or otherwise).
